festim/mesh/mesh_1d.py

```diff
@@ -19,8 +20,8 @@ class Mesh1D(Mesh):
     def __init__(self, vertices, **kwargs) -> None:
         self.vertices = vertices
 
-        mesh = self.generate_mesh()
-        super().__init__(mesh=mesh, **kwargs)
+        self.mesh = self.generate_mesh()
+        super().__init__(mesh=self.mesh, **kwargs)
```

Comment:
the `.mesh` attribute is set in `festim.Mesh`, let's not set it here too:

```suggestion
        mesh = self.generate_mesh()
        super().__init__(mesh=mesh, **kwargs)
```